packages/compiler-cli/src/ngtsc/typecheck/src/api.ts

```diff
@@ -133,6 +133,15 @@ export interface TypeCheckingConfig {
    */
   checkTypeOfDomEvents: boolean;
 
+  /**
+   * Whether to infer the type of local references.
+   *
+   * If this is `true`, the type of any `#ref` variable in the template will be determined by the
```

Comment:
```suggestion
   * If this is `true`, the type of `#ref` variables in the template will be determined by the
```
Mostly, I just think that we should avoid using any and then `any` to avoid confusion here.